plugins/inputs/nfsclient/README.md

```diff
@@ -0,0 +1,39 @@
+# Telegraf plugin: NFSClient
+
+#### Plugin arguments:
+- **fullstat** bool: Collect per-operation type metrics
+
+#### Description
+
+The NFSClient plugin collects data from /proc/self/mountstats, by default it will only include a quite limited set of IO metrics.
+If fullstat is set, it will collect a lot of per-operation statistics.
+
```

Comment:
Can you add the config (yes it's dead simple) to the readme as well as some example output. Probably just need the non fullstat ones:
```
nfsstat_write,mountpoint=/mnt/test,serverexport=192.168.1.100:/exports/test write_ops=0i,write_retrans=0i,write_bytes=0i,write_rtt=0i,write_exe=0i 1549669611000000000
nfsstat_read,mountpoint=/mnt/test,serverexport=192.168.1.100:/exports/test read_ops=877i,read_retrans=0i,read_bytes=113703016i,read_rtt=675626i,read_exe=693656i 1549669611000000000
```

Reply:
Please also address this comment.